tutorials/heritage/index.md

```diff
@@ -0,0 +1,6 @@
+\defgroup tutorial_heritage Heritage tutorials
+\ingroup Tutorials
+\brief Tutorials not anymore in line with times
+
+ROOT is a rapidly evolving toolkit, as well as many of the libraries and languages it relies on.
```

Comment:
That sounds as if tutorials (and ROOT usage) get constantly out of date. Maybe je strike this sentence? The second sentence has all the motivation we need.

Reply:
sure.